src/domain/Progress/ProgressTracker/ProgressTracker.examples.md

```diff
@@ -0,0 +1,38 @@
+The **Progress Tracker** is a visual representation of a users progress through a set of steps.
+They guide the user through a number of steps in order to complete a specified process.
+
+```jsx
+initialState = { value: 1 };
+
+<ProgressTracker currentIndex={state.value}>
+    <ProgressTracker.Step label='Report Issue' onClick={() => setState({value:0})}/>
+    <ProgressTracker.Step label='Initiate Plan' onClick={() => setState({value:1})}/>
+    <ProgressTracker.Step label='Add Cost and  Follow Up' onClick={() => setState({value:2})}/>
+</ProgressTracker>
+```
+
+```jsx
+initialState = { value: 1 };
+
+<ProgressTracker currentIndex={state.value} isMobile>
+    <ProgressTracker.Step label='Report Issue' onClick={() => setState({value:0})}/>
+    <ProgressTracker.Step label='Initiate Plan' onClick={() => setState({value:1})}/>
+    <ProgressTracker.Step label='Add Cost and  Follow Up' onClick={() => setState({value:2})}/>
+</ProgressTracker>
+```
+### Steps
+Progress Tracker require at least two steps, with no more than six.
```

Comment:
`Progress Tracker should only be used for tasks requiring 2 to 6 steps.`

also just add this as the first point in the Best Practices section